### PR TITLE
ensure temp changelog is unique

### PR DESCRIPTION
**Pull request by EndangeredMassa**
Ensure that the temp changelog is unique by appending the current SHA to the filename.

---

Addresses: https://github.com/EndangeredMassa/npub/issues/22

### CODE REVIEW

File: src/publish/index.coffee [hunk 1]
```diff
@@ -53,41 +53,45 @@ module.exports = (dir, log, config, version, testCommand) ->
         changelog.build version, (error, tempChangelog) ->
           endIf(error)
 
-          tempChangelogPath = changelog.write(tempChangelog)
+          git.getSha (error, sha) ->
```
Comment (jkrems) [inline src/publish/index.coffee]: Niiiice! :D

**Comment by jkrems** [comment]
Sounds good to me! :shipit:

**Comment by EndangeredMassa** [comment]
I tested this and it seemed to work very well.

**Comment by EndangeredMassa** [comment]
published in [1.1.0](https://github.com/EndangeredMassa/npub/releases/tag/v1.1.0)

**Pull request merged**